draft-ldbc-cats-framework.md

```diff
@@ -320,47 +320,48 @@ The service metric advertisements are consumed by the C-PS at CATS-Router 1 that
+Figure 2 mainly describes a per-instance computing-related metric distribution. In the case of distributing aggregated per-site computing-related metrics, the per-instance CB-ID information will not be included in the advertisement. Instead, a per-site CB-ID may be used in case multiple sites are connected to the CATS router to explicitly indicate the site the aggregated metrics come from.
 
-However, CB-ID is not needed in the distribution (in above example) if the edge site can support consistently service instance selection. For easy deployment, aggregated per-site computing-related metrics distribution is recommeded.
+However, a CB-ID is not required if the edge site can support consistently service instance selection.
 
-## Service Demand Handling
+## Service Demand Processing
 
-The C-PS determines the best choice of Egress CATS-Router and path to that router given the service and network metrics distributed as described in the previous section. The C-PS may be collocated with the Ingress CATS-Router (as shown in {{fig-cats-example-overlay}}) or the selection function may be offloaded to a dedicated server.
+The C-PS computes and selects paths that lead to CATS-Routers according to the service and network metrics that have been advertised. The C-PS may be collocated with a CATS-Router (as shown in {{fig-cats-example-overlay}}) or logically centralized.
 
-This document does not describe any algorithms for path selection, but it is expected that a service demand or local policy may give directions to the C-PS in terms of "objective functions" that describe the desired behavior from the network and the chosen service instance.
+This document does not specify any algorithm for path computation and selection purposes, but it is expected that a service demand or local policy may feed the C-PS computation logic with Objective Functions that provide some information about the path characteristics (e.g., in terms of maximum latency) and the selected service instance.
 
-In the example in {{fig-cats-example-overlay}}, when the client sends a service demand to CATS-Router 1, the router consults the C-PS to select a service instance at a remote edge site accessed through a particular Egress CATS-Router. The C-PS also determines a path to the Egress CATS-Router. This information is provided to the Ingress CATS-Router (CATS-Router 1) so that it can correctly encapsulate and route packets.
+In the example of {{fig-cats-example-overlay}}, when the client sends a service demand to CATS-Router 1, the router solicits
+the C-PS to select a service instance hosted by an edge site that can be accessed through a particular CATS-Router. The C-PS also determines a path to the CATS-Router. This information is provided to CATS-Router 1 so that it can forward packets to their proper destination, as computed by the C-PS.
```

Comment:
```suggestion
the C-PS to select a service instance hosted by an edge site that can be accessed through a particular CATS-Router. The C-PS also determines a path to the Egress CATS-Router. This information is provided to Ingress CATS-Router (CATS-Router 1) so that it can forward packets to their proper destination, as computed by the C-PS.
```